package.json

```diff
@@ -53,6 +53,7 @@
     "react-router-dom": "^5.1.2",
     "react-scripts": "3.4.1",
     "react-switch": "^5.0.1",
+    "react-test-renderer": "^17.0.1",
```

Comment:
Since this is not being used runtime we can move it to devDependencies.

Reply:
It's not being used anywhere either, I suppose we are adding this for future snapshot use?

Reply:
If we're not planning on using it here, then let's add it in a PR that uses it, to keep things clean/changes easy to follow.